packages/jemalloc/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://jemalloc.net/
+TERMUX_PKG_DESCRIPTION="jemalloc is a general purpose malloc(3) implementation that emphasizes
+fragmentation avoidance and scalable concurrency support."
+TERMUX_PKG_LICENSE="CUSTOM"
```

Comment:
`CUSTOM` should be *lowercase*. I won't change build/upload scripts to understand WhAtEVer CAse license names.

`custom` should be used if package uses self-written license. License of jemalloc is `BSD 2-Clause`.